src/ignitus-Testimonial/constants.js

```diff
@@ -1 +1,3 @@
-export const API_URL = 'https://ignitusrestapi.herokuapp.com/testimonials';
+const API_URL = 'https://ignitusrestapi.herokuapp.com/testimonials';
```

Comment:
Can we create one variable like 
const API_URL = 'https://ignitusrestapi.herokuapp.com in api contatns , get it here and append?

Reply:
yes it's fine @outsourcegb but we will be writing microservices and other restapi as well in the future? although creating approach is also fine but not exporting that one variable as default.

Reply:
FYI, this constant is not being used anywhere inside ignitus-Testimonial, plus I saw this API_URL is repeated among many components without being used (yet). My suggestion is we can put this constant.js at the root folder and use it anywhere inside the app.

So far these components exporting API_URL with the same value:

1. ignitus-Contributors/constant.js
2. ignitus-Introduction/constant.js
3. ignitus-Navigation/constant.js
4. ignitus-GetStarted/constant.js
5. ignitus-Testimonial/constant.js